docs/overview/trustbloc-did-method.md

```diff
@@ -186,60 +254,181 @@ This is a process to update automatically, to reach the current consortium confi
-- If, after that, there is a cached consortium config for the DID's domain, select a sidetree endpoint from a cached stakeholder and perform the sidetree operation.
-
-#### Method Operations
-
-All method operations wrap a [Sidetree operation](https://github.com/decentralized-identity/sidetree/blob/master/docs/protocol.md#sidetree-rest-api), on a sidetree endpoint as determined by the process above.
-
-The operations are summarized here, and described in detail in the link above.
-
-##### Create
-
-A *Create* operation generates and returns a DID document, constructed using the data sent in the POST request.
-
-##### Resolve
-
-A *Resolve* operation fetches a document with a given DID. Note here - the sidetree spec indicates that the request path looks like `[endpoint-path]/[did]`, with the examples all using the DID namespace `did:sidetree`. A sidetree node configured for a trustbloc consortium would instead use a DID namespace of `did:trustbloc:[consortium domain]`.
+### Implementation Notes
+_This section is non-normative_
 
-##### Update
+When implementing a client which performs DID operations in a consortium, it is useful to split out the *read* operation (Resolve) from the *edit* operations (Create, Update, Recover, Deactivate). A resolver does not need any special permissions, and resolution is the most common operation, being needed for any transaction using a DID as an identifier.
 
-An *Update* operation provides a patch to add or remove keys and service endpoints to a particular DID document.
+The edit operations require a client which can securely store key material - it must sign the contents of edit operation messages, and must generate and store one-time passwords for later provision, to prove ownership of the DID.
 
-##### Recover
+### Example Client Flows
+_This section is non-normative_
 
-A *Recover* operation replaces a DID document for a given DID with a new version of the document, in a way that ensures a stolen device cannot update the document using old operation passwords. This therefore recovers control of the DID document.
+This section contains worked examples demonstrating the full process a client takes, to discover endpoints in the consortium and perform a Sidetree operation.
 
-##### Revoke (Delete)
+###### Consortium files:
+The consortium has a config JWS file at `consortium.net/.well-known/did-bloc/consortium.net.json` containing the payload:
+```json
+{
+    "domain": "consortium.net",
+    "policy": {
+        "cache": {"max-age": 2419200},
+        "num-queries": 2,
+        "sidetree": {
+            "hash-algorithm": "SHA256",
+            "key-algorithm": "NotARealAlg2018",
+            "max-encoded-hash-length": 100,
+            "max-operation-size": 8192
+        }
+    },
+    "members": [
+        {
```

Comment:
We may also need a key identifier from the DID.
The did:key should match the key identified by the DID and the identifier.